test/integration/serializers/sample.json

```diff
@@ -31,20 +34,28 @@
                 ]]
             ]]
           }]
-        ]],
+        ]]],
         ["transition", {}, {"relation": "Frob"}, [
-          ["httpTransaction", {"title": "Get a frob", "description": "Gets information about a single frob instance"}, {}, [
-            ["httpRequest", {}, {"method": "GET"}, null],
+          ["httpTransaction", {"title": "Get a frob"}, {}, [
+            ["copy", {}, {}, "Gets information about a single frob instance"],
```

Comment:
This is inconsistent with the diff in [here](https://github.com/apiaryio/fury.js/pull/45/files#r38299525). In that diff, the `copy` element is in `httpRequest`. While here, it is in `httpTransaction`. While I feel like this should be in `transition`.